sensor_stack/sparton/src/sparton.py

```diff
@@ -241,10 +246,71 @@ def getData():
     allData.data[12] = temp
     
     return allData
-    
- 
-if __name__ == '__main__':
-    """    
+
+def getOrientation():
+
+    global roll
+    global pitch
+    global yaw
+
+    quaternion = 
+
+    return quaternion
+
+def getOrientationCovariance():
+
+    # global roll
+    # global pitch
+    # global yaw
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
```

Comment:
@athityakumar Move all of this into the ROS Param system, by writing it in a YAML file. That's the way to go from here on out. No declaration of constants inside code.